src/website/src/pages/ec/components/button/docs/usage.mdx

```diff
@@ -3,39 +3,74 @@ title: Usage
 order: 1
 ---
 
-import { Paragraph } from '@ecl/website-components';
+import { Link, Paragraph } from '@ecl/website-components';
 
 <Paragraph size="lead">
   Use a button to trigger and carry out an action on a page.
 </Paragraph>
 
-## When to use
-
-The ECL has five buttons with different visual designs to make a distinction between the kind of action the button will carry out.
+## Anatomy
 
-| Button name           | Visual style                                                                                            | Usage                                                                                         |
-| --------------------- | ------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------- |
-| Call-to-action button | ![Button Image](https://inno-ecl.s3.amazonaws.com/media/images/EC/Buttons/Button_CTA_Default.png)       | for the most important action on the page like "Apply for funding" or "Watch live streaming". |
-| Primary button        | ![Button Image](https://inno-ecl.s3.amazonaws.com/media/images/EC/Buttons/Button_Primary_Default.png)   | for primary actions in forms, to initiate a search query or to apply a filter.                |
-| Secondary button      | ![Button Image](https://inno-ecl.s3.amazonaws.com/media/images/EC/Buttons/Button_Secondary_Default.png) | or secondary actions.                                                                         |
-| Ghost button          | ![Button Image](https://inno-ecl.s3.amazonaws.com/media/images/EC/Buttons/Button_Ghost_Default.png)     | when an action does not require dominance on a page, for instance to "cancel" a form.         |
-| Search button         | ![Button Image](https://inno-ecl.s3.amazonaws.com/media/images/EC/Buttons/Button_Search_Default.png)    | use exclusively for the search function.                                                      |
+| Name  | Mandatory |
+| ----- | :-------: |
+| Label |     X     |
+| Icon  |           |
 
-Each page can have one to two Primary (or Call-to-action) buttons. Any remaining calls-to-action should be represented as Secondary or Ghost buttons.
+Buttons are defined by 3 major attributes:
 
 ### Button labels
 
-Button labels tell users what will happen when they click the button. Use verbs that describe the action, such as Send, Add or Delete. Do not use capitalization on buttons (only the first word in a phrase and proper nouns are capitalized) and no more than three words for button labels.
+Button labels tell users what will happen when they click the button.
+
+- Should be clear and distinct.
+- Use verbs that describe the action, such as Send, Add or Delete.
+- Keep it short, no more than 3 words.
```

Comment:
```suggestion
- Should be as short as possible, no more than 2 words.
```

Reply:
"keep it short" is shorter than "should be as short as possible" ;-)
Why no more than 2 words? this is very restrictive compared to other design systems

Reply:
Keep it short, no more than 3 words.